core/arch/opnd.h

```diff
@@ -272,6 +272,11 @@ enum {
     DR_REG_Z20, DR_REG_Z21,  DR_REG_Z22,  DR_REG_Z23,
     DR_REG_Z24, DR_REG_Z25,  DR_REG_Z26,  DR_REG_Z27,
     DR_REG_Z28, DR_REG_Z29,  DR_REG_Z30,  DR_REG_Z31,
+
+    DR_REG_P0,  DR_REG_P1,   DR_REG_P2,   DR_REG_P3,
```

Comment:
This breaks binary compatibility as it changes the value of  `DR_REG_NZCV`, etc. below.  (Those `DR_REG_Z*` are also newly inserted (right?) and also break it.)  Please either insert at the end or list this change under the compatibility-breaking part of the changelist in release.dox.